src/display/editor/tools.js

```diff
@@ -145,6 +171,14 @@ class ImageManager {
     return this.getFromUrl(data.url);
   }
 
+  getSVGUrl(id) {
```

Comment:
Maybe a tiny big easier to read as follows (with fewer upper-case letters in a row)?
```suggestion
  getSvgUrl(id) {
```